external/external.sh

```diff
@@ -246,13 +254,32 @@ if [ $command_type == "run" ]; then
 				
 				restore_ready_checkpoint_image="criu-restore-ready-with-jdk"
 				restore_ready_checkpoint_tag="${JDK_VERSION}-${JDK_IMPL}-${docker_os}-${platform}-${node_label_micro_architecture}"
-				tagged_restore_ready_checkpoint_image="${docker_registry_url}/${restore_ready_checkpoint_image}:${restore_ready_checkpoint_tag}"
-				echo "tagged_restore_ready_checkpoint_image is $tagged_restore_ready_checkpoint_image"
-				docker commit --change='ENTRYPOINT ["/bin/bash", "/test_restore.sh"]' $test-test $tagged_restore_ready_checkpoint_image
+				restore_ready_checkpoint_image_fodler="${docker_registry_url}/${restore_ready_checkpoint_image}/${restore_ready_checkpoint_tag}"
 
-				echo "Pushing docker image ${restore_ready_checkpoint_image}:${restore_ready_checkpoint_tag} to docker registry"
-				docker push $tagged_restore_ready_checkpoint_image
+				pipeline_name="other"
+				if [[ $job_name == *"CRIU_Image_Upload"* ]]; then
+					pipeline_name="nightly"
+				elif [[ $job_name == *"Grinder"* ]]; then
+					pipeline_name="grinder"
+				fi
```

Comment:
For easy triage and maintenance, I would prefer to keep the job_name value that was passed in.

Reply:
`job_name` remains the same for docker folder same. Thanks.